src/domain/settings/return-reasons/create-reason-modal.tsx

```diff
@@ -48,51 +61,50 @@ const CreateReturnReasonModal = ({
         <Modal.Header handleClose={handleClose}>
           <span className="inter-xlarge-semibold">Add Reason</span>
         </Modal.Header>
-        <Modal.Content>
-          <div className="flex">
-            <Input
-              ref={register({ required: true })}
-              name="value"
-              label="Value"
-              placeholder="wrong_size"
-            />
+        <form onSubmit={handleSubmit(onCreate)}>
+          <Modal.Content>
+            <div className="flex">
+              <Input
+                {...register("value", { required: true })}
+                label="Value"
+                placeholder="wrong_size"
+              />
+              <Input
+                className="ml-base"
+                {...register("label", { required: true })}
+                label="Label"
+                placeholder="Wrong size"
+              />
```

Comment:
nit: mark these as required
```suggestion
              <Input
                {...register("value", { required: true })}
                label="Value"
                placeholder="wrong_size"
                required
              />
              <Input
                className="ml-base"
                {...register("label", { required: true })}
                label="Label"
                placeholder="Wrong size"
                required
              />
```